email-ballerina/build.gradle

```diff
@@ -0,0 +1,204 @@
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import org.apache.tools.ant.taskdefs.condition.Os
+
+description = 'Ballerina - Ballerina Email Connector'
+
+configurations {
+    jbalTools
+    emailBallerinaBirDir
+    emailBallerinaJar
+    packingJars
+}
+
+dependencies {
+    jbalTools ("org.ballerinalang:jballerina-tools:${ballerinaLangVersion}") {
+        transitive = false
+    }
+    compile project(':email-native')
+    packingJars (group: 'com.sun.mail', name: 'javax.mail', version: "${javaMailVersion}") {
+        transitive = true
+    }
+    packingJars (group: 'com.icegreen', name: 'greenmail', version: "${greenmailVersion}") {
+        transitive = false
+    }
+    packingJars (group: 'org.testng', name: 'testng', version: "${testngVersion}") {
+        transitive = false
+    }
+}
+
+clean {
+    delete "$project.projectDir/target"
+    delete "$project.projectDir/Ballerina.lock"
+}
+
+jar {
+    manifest {
+        attributes('Implementation-Title': project.name,
+                'Implementation-Version': project.version)
+    }
+}
+
+task copyPackingJarsToLib(type: Copy) {
+    into "$project.projectDir/lib"
+    from configurations.packingJars
+}
+
+task unpackJballerinaTools(type: Copy) {
+    configurations.jbalTools.resolvedConfiguration.resolvedArtifacts.each { artifact ->
+        from zipTree(artifact.getFile())
+        into new File("${buildDir}/target/extracted-distributions", "jballerina-tools-zip")
+    }
+}
+
+def moduleName = "email"
+def moduleOrg = "ballerina"
+def ballerinaConfigFile = new File("$project.projectDir/Ballerina.toml")
+def artifactBallerinaDocs = file("$project.projectDir/build/docs_parent/")
+def artifactCacheParent = file("$project.projectDir/build/cache_parent/")
+def artifactLibParent = file("$project.projectDir/build/lib_parent/")
+def tomlVersion = project.version.split("-")[0]
+def ballerinaCentralAccessToken = System.getenv('BALLERINA_CENTRAL_ACCESS_TOKEN')
+def targetBallerinaJar = file("$project.projectDir/target/caches/jar_cache/${moduleOrg}/${moduleName}/${tomlVersion}/${moduleOrg}-${moduleName}-${tomlVersion}.jar")
+def targetNativeJar = file("$project.rootDir/${moduleName}-native/build/libs/${moduleName}-native-${project.version}.jar")
+
+task ballerinaBuild {
+    inputs.dir file(project.projectDir)
+    doLast {
+        def originalConfig = ballerinaConfigFile.text
+        def newConfig = ballerinaConfigFile.text.replace("@project.version@", project.version)
+        newConfig = newConfig.replace("@toml.version@", tomlVersion)
+        ballerinaConfigFile.text = newConfig
+
```

Comment:
Shall we add this configuration here https://github.com/ballerina-platform/module-ballerina-jwt/blob/6f13d4f53f58b65332c2219544130622fad00e96/jwt-ballerina/build.gradle#L114

Reply:
Added.